roles/generate-jenkins/templates/Jenkinsfile.j2

```diff
@@ -562,6 +563,43 @@ pipeline {
              "visibility":"public"}' '''
       }{{ ' ' }}
     }
+    /* #######################
+           Scarf.sh package registry
+       ####################### */
+    // Add package to Scarf.sh and set permissions
+    stage("Scarf.sh package registry"){
+      when {
+        environment name: 'EXIT_STATUS', value: ''
+      }
+      steps{
+        sh '''#! /bin/bash
+              set -e
+              PACKAGE_UUID=$(curl -sX GET -H "Authorization: Bearer ${SCARF_TOKEN}" https://scarf.sh/api/v1/packages | jq -r '.[] | select(.name=="{{ lsio_project_name_short }}/{{ project_name }}") | .uuid')
+              if [ -z "${PACKAGE_UUID}" ]; then
+                echo "Adding package to Scarf.sh"
+                PACKAGE_UUID=$(curl -sX POST https://scarf.sh/api/v1/packages \
+                  -H "Authorization: Bearer ${SCARF_TOKEN}" \
+                  -d '{"name":"{{ lsio_project_name_short }}/{{ project_name }}",\
+                       "shortDescription":"example description",\
```

Comment:
Is this meant to have a variable?

Reply:
Not yet. All the current ones have that as the description. And it's not really listed on their website publicly so no point in setting it properly just yet. We can tackle it later if need be.